app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -57,12 +59,21 @@ sealed class RouteAction(
         object FaqList : AppWebPage(
             Constant.Faq.uri,
             R.string.nav_menu_faq,
-            TelemetryEventObject.settings_faq)
+            TelemetryEventObject.settings_faq
+        )
 
         object SendFeedback : AppWebPage(
             Constant.SendFeedback.uri,
             R.string.nav_menu_feedback,
-            TelemetryEventObject.settings_provide_feedback)
+            TelemetryEventObject.settings_provide_feedback
+        )
+    }
+
+    sealed class Onboarding(
+        eventObject: TelemetryEventObject
+    ) : RouteAction(TelemetryEventMethod.show, eventObject) {
+        object FingerprintAuth : Onboarding(TelemetryEventObject.onboarding_fingerprint)
```

Comment:
This could even just be named `Fingerprint` IMO

Reply:
followup musing, no need to address: if Android adds face recognition, we will need to change all of these to Biometric like iOS 😱

Reply:
I was thinking about that at first, which is why I had such ragtag naming for a while (fingerprint, fingerprint auth, biometric fingerprint, etc). I think it's fine to have consistent `FingerprintOnboarding` now and then we can change it later 🤔